### ISSUE TITLE
Respawns

### ISSUE THREAD

**Issue by LordVector** [MaccaNZ97/NZ-Expansion-Cherno#1]
I've used your config in my server but after attaching the files I've found that my Bed Respawning mod stopped allowing players to respawn on there beds. Is there something you've added in your configs to override the default nature of respawns in the Expansion mod? I've been trying to fix this with the standard settings but no luck so far.

**Comment by LordVector** [comment]
So I did have Bed respawns working while using the Expansion Bundle and Licensed but then I found your files and installed the whole package. After doing that is when most of my already configured files got changed or reverted to defaults. I've toggled SpawnOnTerritory off and no luck. Wasn't sure if there's anything in your code that I should look for to change this. Sounds like that's a no then from you, Correct?

**Comment by MaccaNZ97** [comment]
If you had it working then you should revert Territories' and spawn files back to how they were before. TBH I wouldnt use anyone else's config verbatim use it as a way how its configured or try individual items only.

Also I have built Macca 2.0 which is more rock solid  https://github.com/MaccaNZ97/Macca-Expansion-Dayz-Server

Again use what you need but you need to cater to your instance.

**Comment by LordVector** [comment]
Yeah I kinda did that in reverse the first time lol. New to hosting DayZ. Grabbed your config and edited to my liking after that. Of course now that I have all that configed I'll definitely do it the slower more precise way next time.
